docs/_docs/user-guide/terminology.md

```diff
@@ -8,36 +8,36 @@ permalink: user-guide/terminology
 
 ## Terminology
 
-Throughout the documentation terms like "agent" or "implant" are used to reference various components of Realm. Below we attempt to define some of those terms, to add some clarity to that other documentation.
-
-### Host
-
-A Host is a system that is in-scope for the current engagement. It is used to establish a logical boundary between different systems in an engagement (e.g. between a webserver and a database). This enables operations to target a particular system, for example you may want to list files on a web server in your engagement scope.
+Throughout the documentation terms that are used to reference various components of Realm. Below we attempt to define some of those terms, to add some clarity to that other documentation.
 
 ### Implant
 
 References malicious code or persistence mechanisms that are deployed to compromise target systems. Imix is the primary implant used with Realm.
 
 ### Agent
 
-An Agent is a type of implant which retrieves execution instructions by connecting to our backend infrastructure (calling back) and querying for new tasks.
+An agent is a compiled program.
```

Comment:
I think we need to expand on this not sure if we wanna do that in this PR

Reply:
Should I leave it as it was or keep the change that we discussed?

Reply:
This simple sentence what what we came up with about 2hrs ago. I was asking if we should revert back to the expansive text.